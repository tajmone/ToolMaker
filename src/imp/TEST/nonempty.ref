  OK 1
  OK 2
  OK 3
  OK 4
  OK 5
  OK 6
%%IF (C(*))
%%IF (C(*))         * * * FACIT * * *
%%ENDIF
%%IF ({1}[2](*))
%%IF ({1}[2](*))    * * * FACIT * * *
%%ENDIF
%%IF (<NULL>(*))
%%IF (<NULL>(*))    * * * FACIT * * *
%%ENDIF
%%IF (%%(C)(*))
%%IF (%%(C)(*))     * * * FACIT * * *
%%ENDIF
