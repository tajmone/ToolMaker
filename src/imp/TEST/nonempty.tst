%%SET A ()
%%SET B (1)
%%IF (A (*))
  Fel 1
%%ELSE
  OK 1
%%ENDIF
%%IF (B (*))
  OK 2
%%ELSE
  Fel 2
%%ENDIF
%%IF ({1} (*))
  OK 3
%%ELSE
  Fel 3
%%ENDIF
%%IF ({} (*))
  Fel 4
%%ELSE
  OK 4
%%ENDIF
%%IF ({1,2,3}[2] (*))
  OK 5
%%ELSE
  Fel 5
%%ENDIF
%%IF ({1,2,{},4}[3] (*))
  Fel 6
%%ELSE
  OK 6
%%ENDIF
%%IF (C (*))
`%%IF (C(*))         * * * FACIT * * *
%%ENDIF
%%IF ({1}[2] (*))
`%%IF ({1}[2](*))    * * * FACIT * * *
%%ENDIF
%%IF (%%(A) (*))
`%%IF (<NULL>(*))    * * * FACIT * * *
%%ENDIF
%%IF (%%(C) (*))
`%%IF (`%%(C)(*))     * * * FACIT * * *
%%ENDIF
