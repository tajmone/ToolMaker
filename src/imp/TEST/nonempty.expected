########## nonempty ##########
  OK 1
  OK 2
  OK 3
  OK 4
  OK 5
  OK 6
%%IF (C(*))
%%IF (C(*))         * * * FACIT * * *
%%ENDIF
%%IF ({1}[2](*))
%%IF ({1}[2](*))    * * * FACIT * * *
%%ENDIF
%%IF (<NULL>(*))
%%IF (<NULL>(*))    * * * FACIT * * *
%%ENDIF
%%IF (%%(C)(*))
%%IF (%%(C)(*))     * * * FACIT * * *
%%ENDIF


nonempty.tst

   33.  %%IF (C (*))
=====>        1

  *1*   204 W : Variable "C" not set.

   36.  %%IF ({1}[2] (*))
=====>            1

  *1*   209 E : Index value "2" out of range.

   39.  %%IF (%%(A) (*))
=====>        1

  *1*   214 E : Empty string not allowed as variable name
  *1*   204 W : Variable "<NULL>" not set.

   42.  %%IF (%%(C) (*))
=====>        1  2

  *1*   204 W : Variable "%%(C)" not set.
  *2*   204 W : Variable "C" not set.


2 error(s) detected.
4 warning(s) issued.

